Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'test-unit', '~> 3.2'
 
 gem 'bundler'
 
-gem "rails", "~> 5.2.5"
+gem "rails", "~> 5.2.6"
```

Comment:
Can you change this to `~> 5.2`? That way Dependabot can automatically bump patch versions later on.